pages/_admin/nav.tsx

```diff
@@ -0,0 +1,82 @@
+import { ReactNode } from 'react';
+import {
+  Box,
+  Flex,
+  HStack,
+  Link,
+  IconButton,
+  useDisclosure,
+  useColorModeValue,
+  Stack,
+} from '@chakra-ui/react';
+import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
+
+import { signIn, signOut } from 'next-auth/client'
+
+const Links = [
+  {href:'/_admin/', text:'Admin'},
+  {href:'/_admin/?deleted=true', text:'Show Deleted'},
```

Comment:
There's probably a better way to do this.

Reply:
Seems reasonable to me!